docs/docs/externalpayloadstorage.md

```diff
@@ -84,3 +84,29 @@ Set the following properties to the desired values in the JVM system properties:
 | workflow.external.payload.storage.azure_blob.task_output_path | Path prefix where tasks output will be stored with an random UUID filename | task/output/ |
 
 The payloads will be stored as done in [Amazon S3](https://github.com/Netflix/conductor/blob/master/core/src/main/java/com/netflix/conductor/core/utils/S3PayloadStorage.java#L149-L167).
+
+### PostgreSQL Storage
+
+Frinx provides an implementation of [PostgreSQL Storage](https://www.postgresql.org/) used to externalize large payload storage.
```

Comment:
What is `Frinx`?

Reply:
This is the firm on whose behalf I added this feature. 
https://frinx.io/